code/datums/wires/_wires.dm

```diff
@@ -317,9 +317,18 @@
 			I = L.is_holding_tool_quality(TOOL_MULTITOOL)
 			if(I || isAdminGhostAI(usr))
 				if(I && holder)
-					I.play_tool_sound(holder, 20)
-				pulse_color(target_wire, L)
-				. = TRUE
+				//SKYRAT EDIT CHANGE BEGIN
+					if(istype(I, /obj/item/multitool))
+						var/obj/item/multitool/the_tool = I
+						var/datum/component/cell/cell_component = the_tool.GetComponent(/datum/component/cell)
+						if(cell_component.simple_power_use(L, the_tool.power_cell_use))
```

Comment:
May be retarded, but doesn't this assume that there IS in fact a cell in the multitool?

Reply:
Simple power use will return and inform the user of why it has failed the checks, it's all handled in simple_power_use, makes life easy